docs/index.rst

```diff
@@ -22,12 +22,12 @@ MTF has a presence on the following websites:
 
 * `Documentation`_ is available on ReadTheDocs.
 * A `Package repository`_ is available on Fedora Copr.
-* `MTF's code`_ and the issue tracker for sharing bugs and feature ideas are stored on Pagure.
+* `MTF's code`_ and the issue tracker for sharing bugs and feature ideas are stored on GitHub.
 
-.. _Documentation: http://modularity-testing-framework.readthedocs.io
+.. _Documentation: http://meta-test-family.readthedocs.io
 .. _a modular Fedora: https://docs.pagure.org/modularity/
 .. _Package repository: https://copr.fedorainfracloud.org/coprs/phracek/Modularity-testing-framework/
```

Comment:
This is wrong. Please use https://copr.fedorainfracloud.org/coprs/phracek/meta-test-family/